docs/quickstarts/6_aspnet_identity.rst

```diff
@@ -2,180 +2,64 @@
 Using ASP.NET Core Identity
 ===========================
 
+In the prior quickstarts, the users of the application were modeled with hard-coded, in-memory usernames and passwords.
 IdentityServer is designed for flexibility and part of that is allowing you to use any database you want for your users and their data (including passwords).
-If you are starting with a new user database, then ASP.NET Identity is one option you could choose.
+If you are starting with a new user database, then ASP.NET Identity is one option you could choose to manage your users' identity information.
 This quickstart shows how to use ASP.NET Identity with IdentityServer.
 
-This quickstart assumes you've been through all of the prior quickstarts. 
-The approach this quickstart takes to using ASP.NET Identity is to create a new project from the ASP.NET Identity template in Visual Studio.
-This new project will replace the prior IdentityServer project we built up from scratch in the previous quickstarts.
+To use ASP.NET Identity, a new project will be created to replace the "QuickstartIdentityServer" that was used in the prior quickstarts.
+You will use the "IdentityServer with ASP.NET Identity" template to create the new project.
 All the other projects in this solution (for the clients and the API) will remain the same.
 
-New Project for ASP.NET Identity
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-
-The first step is to add a new project for ASP.NET Identity to your solution.
-Given that a lot of code is required for ASP.NET Identity, it makes sense to use the template from Visual Studio.
-You will eventually delete the old project for IdentityServer (assuming you were following the other quickstarts), but there are several items that you will need to migrate over (or rewrite from scratch as described in the prior quickstarts).
+Install the IdentityServer Templates
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-Start by creating a new "ASP.NET Core Web Application" project.
+If you haven't already, you will need to install the IdentityServer templates.
+They can be installed by running a ``dotnet`` command from the command line::
 
-.. image:: images/6_new_web_project.png
+    dotnet new -i IdentityServer.Templates
```

Comment:
This should be: dotnet new -i IdentityServer4.Templates

Reply:
good catch, thanks!